apps/marsha/templates/app/secret_cloudfront_private_key.yml.j2

```diff
@@ -1,4 +1,3 @@
-{% if MARSHA_VAULT.DJANGO_CLOUDFRONT_PRIVATE_KEY is defined %}
```

Comment:
Maybe I should rollback this to make `DJANGO_CLOUDFRONT_PRIVATE_KEY` optional un Marsha's vault? Ping @lunika